main.py

```diff
@@ -1,7 +1,34 @@
 import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
```

Comment:
Add one blank line between different import groups to follow PEP-8 in the future